onebusaway-android/src/androidTest/java/org/onebusaway/android/io/test/BikeStationRequestTest.java

```diff
@@ -0,0 +1,26 @@
+package org.onebusaway.android.io.test;
+
+import org.onebusaway.android.app.Application;
+import org.onebusaway.android.io.request.bike.OtpBikeStationRequest;
+import org.onebusaway.android.io.request.bike.OtpBikeStationResponse;
+import org.onebusaway.android.mock.MockRegion;
+import org.opentripplanner.routing.bike_rental.BikeRentalStation;
+
+/**
+ * Created by carvalhorr on 7/14/17.
+ */
+public class BikeStationRequestTest extends ObaTestCase {
+
+    public void testGetBikeStationsTampa() {
+        Application.get().setCurrentRegion(MockRegion.getTampa(getContext()), false);
+        OtpBikeStationRequest request = OtpBikeStationRequest.newRequest(getContext(), null, null);
+        OtpBikeStationResponse response = request.call();
+        assertNotNull(response);
+        assertNotNull(response.stations);
+        assertEquals(133, response.stations.size());
+        for(BikeRentalStation station: response.stations) {
```

Comment:
This test is a good start - I'd also get a reference to the first element in the list and assert that each of the fields in the raw JSON match what you pull out of the `station` object.

~~~

      "id": "bike_3566",
      "name": "B-1165",
      "x": -82.40730666666667,
      "y": 28.066505,
      "bikesAvailable": 1,
      "spacesAvailable": 0,
      "allowDropoff": false,
      "isFloatingBike": true,
      "networks": [
        "default"
      ],
      "realTimeData": true
    },
~~~

That way you can make sure Jackson is deserializing all the fields correctly, and that you're not missing any (like the new `isFloatingBike`.